qiskit/transpiler/passes/basis/unroll_custom_definitions.py

```diff
@@ -60,9 +60,9 @@ def run(self, dag):
         if self._basis_gates is None and self._target is None:
             return dag
 
+        basic_insts = {"measure", "reset", "barrier", "snapshot", "delay", "store"}
         if self._target is None:
-            basic_insts = {"measure", "reset", "barrier", "snapshot", "delay", "store"}
-            device_insts = basic_insts | set(self._basis_gates)
+            basic_insts |= set(self._basis_gates)
```

Comment:
The terminology here has gone a bit wrong I think, since `basic_insts |= set(self._basis_gates)` is not the basic instructions anymore - it's the device instructions.  Probably just replace the name `basic_insts` with `device_insts` and it's fine.

Reply:
fixed in 8057595696c1d68a867ab383764391c8847492bd